denizen_scripts/survival/repo-link/death_location.dsc

```diff
@@ -4,7 +4,6 @@ player_death_event:
   events:
     on player death:
       - flag <player> player_death_location:<player.location>
-      
       - teleport <player> spawn
```

Comment:
safety in events

Reply:
i'd also recommend using `player spawns` event over an immediate death take-away